recipes/apu-label-propagation/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "1.0" %}
+
+package:
+  name: apu-label-propagation
+  version: {{ version }}
+
+source:
+  url: https://github.com/anuprulez/apu_label_propagation/archive/refs/tags/1.0.tar.gz
+  sha256: 71d4fb4f37f50a366a629f61b2866f54a5ea06301462d142c65f8170f1dfa96a
+
+build:
+  number: 0
+  skip: True
```

Comment:
This means you are skipping the build. So nothing is happening. Please remove this. Then you need to add the GCC compile command to build.sh

Reply:
Not sure GCC command would be needed. I am providing the compiled C code already in the package repository.

Reply:
As the next step, I will try building the compiled code in the `build.sh`.